.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+# Taken from https://github.com/ssato/python-anyconfig/blob/next/.github/workflows/tests.yml
+# https://github.com/ssato/python-anyconfig/blob/next/LICENSE.MIT
+name: make
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      max-parallel: 5
+      matrix:
+        platform:
+          - ubuntu-latest
+        python-version:
+          - 3.6
+          - 3.7
+          - 3.8
+          - 3.9
```

Comment:
```suggestion
          - 3.9
          - 3.10
          - 3.11
```

Reply:
As far as trying locally, testing on 3.10 failed.
That on 3.11 may fail, too.
I would like to focus on migration. 
Can I add 3.10 and 3.11 later?

Reply:
Sure!